layouts/shortcodes/ecosystem/registry/search-form.html

```diff
@@ -92,6 +92,17 @@
   </form>
 </div>
 
+<div style="text-align: center; margin-top: 20px;">
+  <button 
+      id="scrollToTopBtn" 
+      class="btn rounded-circle position-fixed" 
+      style="background-color: #007bff; color: white; bottom: 40px; right: 40px; display: none; box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2); z-index: 1000;" 
+      aria-label="Scroll to top" 
+      title="Scroll to top"> 
+      <i class="fas fa-arrow-up"></i>
+  </button>
+</div>
```

Comment:
Move the styling into the `_registry.scss` file.

Reply:
Fixed @chalin . Please give another look.